### ISSUE TITLE
Navigation section: Allow for non-link labels

### ISSUE THREAD

**Issue by mattsharkey** [web-illinois/toolkit#120]
The aria-label of the navigation section dropdown toggle is based on the content of the link in the label slot of the component. In order to accommodate sections without a label link, the aria-label should be based on the text content of any slotted content.
